portfolio/src/main/webapp/script.js

```diff
@@ -77,3 +77,100 @@ function createListElement(text) {
+  ['Oxford', 51.7520, -1.2577, 'May-June 2019', 'For the beginning of summer after my freshman year, I studied Computing Ethics at Oxford through the Foundation Fellowship\'s Maymester Program']
+];
+
+/** Places I have visited. */
+var placesVisited = [
+  // Domestic
+  ['New York', 40.7128, -74.0060, 2019],
+  ['Washington D.C.', 38.9072, -77.0369, 2019],
+  ['San Francisco', 37.7749, -122.4194, 2014],
+  ['Los Angeles', 34.0522, -118.2437, 2013],
+  ['Bonita Springs', 26.3398, -81.7787, 2020],
+  ['San Juan', 18.4655, -66.1057, 2020],
+  ['Vieques', 18.1263, -65.4401, 2020],
+  ['Niagra Falls', 43.0962, -79.0377, 2008],  //place holder
+  ['St. Louis', 38.6270, -90.1994, 2012],  // place holder
+  ['Omaha', 41.2565, -95.9345, 2012],  // place holder
+  ['Sioux City', 42.4963, -96.4049, 2012],  // place holder
+  ['Des Moines', 41.5868, -93.6250, 2012],  // place holder
+  ['Lincoln', 40.8136, -96.7026, 2012],  //place holder
+  ['Wichita', 37.6872, -97.3301, 2012],  // place holder
+  ['Miami', 25.7617, -80.1918, 2012],  // place holder
+  ['Barcelona', 41.3851, 2.1734, 2018], 
+  ['Paris', 48.8566, 2.3522, 2019], 
+  ['Split', 43.5081, 16.4402, 2019],
+  ['London', 51.5074, 0.1278, 2019],
+  ['Venice', 45.4408, 12.3155, 2017],
+  ['Prague', 50.0755, 14.4378, 2017],
+  ['Cesky Krumlov', 48.8127, 14.3175, 2017],
+  ['Vienna', 48.2082, 16.3738, 2017],
+  ['Salzburhg', 47.8095, 13.0550, 2017],
+  ['Jerusalem', 31.7683, 35.2137, 2014],
+  ['Tel Aviv', 32.0853, 34.7818, 2014],
+  ['Ayia Napa', 34.9923, 34.0140, 2018]
+]
+
+/** Creates a map and adds it to the page. */
+function createMap() {
+
+  // creates map and adds it to page
+  const map = new google.maps.Map(
+    document.getElementById('map'),
+    {center: {lat: 37.601187, lng: -40.705303}, zoom: 3});
+  
+  // adds lived markers to the map
+  var livedIcon = {
+    url: "/images/livedPin.svg",
+    scaledSize: new google.maps.Size(30, 48),
+    origin: new google.maps.Point(0, 0),
+    anchor: new google.maps.Point(15, 48)};
+
+  for (var i = 0; i < placesLived.length; i++) {
+    var place = placesLived[i];
+    var marker = new google.maps.Marker({
+      position: {lat: place[1], lng: place[2]},
+      map: map,
+      title: (place[0] + ': lived from ' + place[3]),
+      icon: livedIcon
+    });
+    attachWindow(marker, place[4], place[0], place[3]);
+  }
+
+  // add visited markers to the map
+  var visitedIcon = {
+    url: "/images/visitedPin.svg",
+    scaledSize: new google.maps.Size(20, 32),
+    origin: new google.maps.Point(0, 0),
+    anchor: new google.maps.Point(10, 32)};
+
+  for (var i = 0; i < placesVisited.length; i++) {
+    var place = placesVisited[i];
+    var marker = new google.maps.Marker({
+      position: {lat: place[1], lng: place[2]},
+      map: map,
+      title: (place[0] + ': visited in ' + place[3]),
+      icon: visitedIcon
+    });
+  }
```

Comment:
These 2 snippets of code are almost identical which makes me feel like they should be factored out into a function.